src/web/components/ApiKeyManagement/KeyEditDialog.tsx

```diff
@@ -82,8 +82,8 @@ function KeyEditDialog({
                   <li key={`Role-${role.id}`}>{role.externalName}</li>
                 ))}
               </ul>
-              If you remove any of these roles from the key, you will not be able to undo this
-              action.
+              If you remove any of these permission(s) from the key, you will not be able to undo
```

Comment:
I think "permissions" makes more sense than "permission(s)" here.

Reply:
Ah yeah that's fair